src/app.js

```diff
@@ -1 +1,41 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { compareNumbers } = require('./compareNumber');
+const { generateNumber } = require('./generator');
+const { terminal } = require('./terminal');
+
+const generatedNumber = generateNumber();
+
+const checkUserNumber = (userNumber) => {
```

Comment:
rely on task, you need to move it in separate module

![image](https://user-images.githubusercontent.com/122156399/234842144-582798a1-9a27-4cf0-bbc4-07cca9b9ceea.png)